src/js/base/settings.js

```diff
@@ -125,6 +125,8 @@ $.summernote = $.extend($.summernote, {
     maxTextLength: 0,
     blockquoteBreakingLevel: 2,
     spellCheck: true,
+    placeholder: null,
+    inheritPlaceholder: false,
```

Comment:
this option is hard to understand

Reply:
These variables are ‘default’ values as like as others in `settings.js`.